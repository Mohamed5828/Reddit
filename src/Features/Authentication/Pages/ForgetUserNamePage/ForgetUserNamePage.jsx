import { PageContainer, Photo, LogInContainer } from "./ForgetUserNamePage.styled";
import ForgetUserName from "Features/Authentication/Layouts/ForgetUserName/ForgetUserName";


/**
 * ForgetUserNamePage component (The page of forget userName)
 * @returns {React.Component}  ForgetUserNamePage component (The page of forget userName)
 */
const ForgetUserNamePage = () => {
  return (
    <PageContainer>
      <Photo></Photo>

      <LogInContainer>
      <ForgetUserName></ForgetUserName>
      </LogInContainer>
    </PageContainer>
  );
};

export default ForgetUserNamePage;
