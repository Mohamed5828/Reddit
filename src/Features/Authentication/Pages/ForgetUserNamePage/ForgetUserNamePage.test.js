import { shallow } from "enzyme";

import ForgetUserNamePage from "./ForgetUserNamePage";

describe("ForgetUserNamePage Page", () => {
  it("this is a test for ForgetUserNamePage Page", () => {
    expect(shallow(<ForgetUserNamePage />)).toMatchSnapshot();
  });
});
