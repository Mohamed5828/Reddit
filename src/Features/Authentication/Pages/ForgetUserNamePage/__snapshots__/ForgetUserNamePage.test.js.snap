// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForgetUserNamePage Page this is a test for ForgetUserNamePage Page 1`] = `
<styled.div>
  <styled.div />
  <styled.div>
    <ForgetUserName />
  </styled.div>
</styled.div>
`;
